dev-support/docker/pkglist

```diff
@@ -33,7 +33,22 @@ openjdk-8-jdk
 python3-setuptools
 python3-pip
 python3.6
+python3.6-dev
+python3.6-venv
 python3.7
+python3.7-dev
+python3.7-distutils
+python3.7-venv
 python3.8
+python3.8-dev
+python3.8-distutils
+python3.8-venv
+python3.9
+python3.9-dev
+python3.9-distutils
+python3.9-venv
 tox
 docker.io
+
+
+python3.6 python3.6-dev python3.6-venv python3.7 python3.7-dev python3.7-distutils python3.7-venv python3.8 python3.8-dev  python3.8-distutils python3.8-venv
```

Comment:
what is this line?

Reply:
I think there is some 'dev' environment, and this file lists all apt packages to install in that environment.

Reply:
ah looks like the line is a leftover, thanks for catching.

Reply:
re: dev envirornment - this line is referenced from `./local-env-setup.sh` (configures current system), and `./start-build-env.sh` (builds and start a development container)